diff -up bind-9.15.3/bin/named/Makefile.in.link bind-9.15.3/bin/named/Makefile.in
--- bind-9.15.3/bin/named/Makefile.in.link	2019-08-22 16:01:35.839689861 +0200
+++ bind-9.15.3/bin/named/Makefile.in	2019-08-22 16:14:34.637830280 +0200
@@ -57,7 +57,7 @@ CDEFINES =      @CONTRIB_DLZ@
 
 CWARNINGS =
 
-DNSLIBS =	../../lib/dns/libdns.@A@ ${MAXMINDDB_LIBS} @DNS_CRYPTO_LIBS@
+DNSLIBS =	../../lib/dns/libdns.@A@ ${MAXMINDDB_LIBS} @DNS_CRYPTO_LIBS@ -ldl
 ISCCFGLIBS =	../../lib/isccfg/libisccfg.@A@
 ISCCCLIBS =	../../lib/isccc/libisccc.@A@
 ISCLIBS =	../../lib/isc/libisc.@A@ ${OPENSSL_LIBS} ${JSON_C_LIBS} ${LIBXML2_LIBS} ${ZLIB_LIBS}
