diff -up bind-9.13.6/bin/named/Makefile.in.link bind-9.13.6/bin/named/Makefile.in
--- bind-9.13.6/bin/named/Makefile.in.link	2019-02-11 22:03:39.842826521 +0100
+++ bind-9.13.6/bin/named/Makefile.in	2019-02-11 22:04:03.227183176 +0100
@@ -52,7 +52,7 @@ CDEFINES =      @CONTRIB_DLZ@
 
 CWARNINGS =
 
-DNSLIBS =	../../lib/dns/libdns.@A@ @DNS_CRYPTO_LIBS@
+DNSLIBS =	../../lib/dns/libdns.@A@ @DNS_CRYPTO_LIBS@ -ldl
 ISCCFGLIBS =	../../lib/isccfg/libisccfg.@A@
 ISCCCLIBS =	../../lib/isccc/libisccc.@A@
 ISCLIBS =	../../lib/isc/libisc.@A@ @OPENSSL_LIBS@
