diff -up bind-9.13.6/bin/named/unix/os.c.prctl_set_dumpable.droplet bind-9.13.6/bin/named/unix/os.c
--- bind-9.13.6/bin/named/unix/os.c.prctl_set_dumpable.droplet	2019-02-11 22:04:19.321428633 +0100
+++ bind-9.13.6/bin/named/unix/os.c	2019-02-11 22:05:05.790137311 +0100
@@ -184,6 +184,10 @@ linux_initialprivs(void) {
 
 	linux_setcaps(caps);
 
+	/* allow us to continue to be able to generate coredumps after setuid:
+	 */
+	prctl(PR_SET_DUMPABLE,1,0,0,0);
+
 	FREE_CAP;
 }
 
