diff -up bind-9.17.6/Makefile.top.omv~ bind-9.17.6/Makefile.top
--- bind-9.17.6/Makefile.top.omv~	2020-10-25 11:19:40.444503049 +0100
+++ bind-9.17.6/Makefile.top	2020-10-25 11:19:50.415630236 +0100
@@ -94,8 +94,5 @@ LIBBIND9_CFLAGS = \
 LIBBIND9_LIBS = \
 	$(top_builddir)/lib/bind9/libbind9.la
 
-LIBLTDL_CFLAGS = \
-	-I$(top_srcdir)/libltdl
-
 LIBLTDL_LIBS = \
-	$(top_builddir)/libltdl/libltdlc.la
+	-lltdl
