diff -up bind-9.12.2-P1/bin/named/Makefile.in.link bind-9.12.2-P1/bin/named/Makefile.in
--- bind-9.12.2-P1/bin/named/Makefile.in.link	2018-08-22 23:08:21.458876126 +0200
+++ bind-9.12.2-P1/bin/named/Makefile.in	2018-08-22 23:08:43.115013802 +0200
@@ -52,7 +52,7 @@ CDEFINES =      @CONTRIB_DLZ@ @USE_PKCS1
 
 CWARNINGS =
 
-DNSLIBS =	../../lib/dns/libdns.@A@ @DNS_CRYPTO_LIBS@
+DNSLIBS =	../../lib/dns/libdns.@A@ @DNS_CRYPTO_LIBS@ -ldl
 ISCCFGLIBS =	../../lib/isccfg/libisccfg.@A@
 ISCCCLIBS =	../../lib/isccc/libisccc.@A@
 ISCLIBS =	../../lib/isc/libisc.@A@ @ISC_OPENSSL_LIBS@
